tests/10apidoc/34room-messages.pl

```diff
@@ -138,10 +138,10 @@ sub matrix_send_room_text_message
       matrix_send_room_text_message( $user, $room_id,
          body => "Here is the message content",
       )->then( sub {
-         matrix_sync($user)
+         matrix_sync( $user )
       })->then( sub {
          my ( $sync_body ) = @_;
-         my $token = $sync_body->{next_batch};
+         my $token = $sync_body->{rooms}->{join}->{$room_id}->{timeline}->{next_batch};
```

Comment:
`next_batch` doesn't exist here (at least in the spec), so I suspect this PR isn't changing anything...

s/next_batch/prev_batch.

(likewise below)

Reply:
Gah, indeed. Weirdly enough, SyTest doesn't complain about that, nor does Synapse by being given an invalid token (which I'd expect it to do). /me fixes